### DISCUSSION TITLE
Understanding Rollups (From A Third Party Source "WhiteBoard Crypto" YouTube Channel)

### DISCUSSION THREAD

**Discussion by ThisIsSaadBinAnis** [Cyfrin/foundry-full-course-cu#1935] [General]
Yesterday I watched the newly added videos associated with ZKSync, Not gonna lie it was a bit hard for me to learn from someone else other than Patrick. Even though I'm confident that Ciara is a fantastic instructor but maybe it was because of my unused braincells that I wasn't able to quite comprehend her way of explaining.

So I searched on youtube about "Rollups" and found a great video by "Whiteboard Crypto" which was exactly what a beginner like me needed. For other people like me I'd leave the video link here so that it can help them as well. 

**What are Rollups in Crypto? ZKSnarks vs Optimistics Rollups Explained**

https://youtu.be/6_nOYsvXMsE?si=CAl0NKALowDWuw3B

**Comment by EngrPips** [comment]
Thank you so much for sharing this resource with us @ThisIsSaadBinAnis.

**Comment by ciaranightingale** [comment]
Hi @ThisIsSaadBinAnis thank you for your feedback!
I would love to hear what made Whiteboard Crypto's video explanation easier for you to understand as a beginner
At Cyfrin, we always love to hear your pointers so that we can keep leveling up the content so that you have the best possible learning experience.

**Reply by ThisIsSaadBinAnis** [reply]
Hey @ciaranightingale hope you're doing great.

I think there might have been few reasons because of which I didn't properly understand the way you explained.

First of all let's address the elephant in the room, I'm not a native english speaker
but I was confident that I've a good grasp on the language yet British English always makes me think twice,
so there was language barrier from my side, actually it might have been the terminology problem.
It was a bit technical and maybe I was having a hard time visualising the concept.

Because it was beginner level video I felt it could've been a bit more metaphorical which is one of the reason "Whiteboard Crypto's" video was very easy for me to understand. 

Another thing I feel like is when "cyfrin" will reshoot their videos 
it would be more engaging and easy to pick up for beginners if the videos have enough motion-graphicsand real life metaphor so that they can connect their thoughts with the digital and the physical.
I think it's important to have these in videos made for beginners.
I'm sure I didn't need to say that because it might be already in the mind of the production house.

And I know that while shooting these courses everything can't go as planned. So it is only natural to have some issues here and there even when the provider is aware of it.

I hope you've an amazing rest of your day.

**Reply by EngrPips** [reply]
Well said, Friend.
